### PR TITLE
Create own section for `explorer-backend`

### PR DESCRIPTION
**Pull request by tdroxler**
The explorer-backend wiki stuff were blended into the full node section.

### CODE REVIEW

**Comment by polarker** [comment]
Maybe we should move it to the "Infrastructure" category. I don't want to introduce too many categories, it's not reader-friendly.

**Comment by nop33** [comment]
Don't forget to set some redirect rules when moving pages: https://github.com/alephium/docs/blob/master/docusaurus.config.js#L167

**Comment by tdroxler** [comment]
Finally get back to this, I moved `explorer-backend` to `infrastructure`

File: docusaurus.config.js [hunk 1]
```diff
@@ -213,7 +213,7 @@ const config = {
           },
           {
             from: "/Explorer-Backend-Starter-Guide.html",
-            to: "/full-node/explorer-backend",
```
Comment (nop33) [inline docusaurus.config.js:216]: There should also be a new redirect rule from `/full-node/explorer-backend` to the new endpoint `/infrastructure/explorer-backen` otherwise we'll have broken links.

File: docusaurus.config.js [hunk 2]
```diff
@@ -213,7 +213,7 @@ const config = {
...
+            to: "/infrastructure/explorer-backend",
```
Comment (tdroxler) [inline docusaurus.config.js:216]: @nop33 it's here :) right next to you

Comment (tdroxler) [reply docusaurus.config.js:216]: image

Comment (nop33) [reply docusaurus.config.js:216]: This is not what I mean. Here you changed an existing redirect to a new one.

What I mean is that we need both:
`/Explorer-Backend-Starter-Guide.html` => `/infrastructure/explorer-backend`
`/full-node/explorer-backend` => `/infrastructure/explorer-backend`

otherwise the original redirect breaks. The `/full-node/explorer-backend` will return a 404 after the change you made.

Comment (nop33) [reply docusaurus.config.js:216]:
```ts
{
            from: "/Explorer-Backend-Starter-Guide.html",
            to: "/infrastructure/explorer-backend",
},
{
            from: "/full-node/explorer-backend",
            to: "/infrastructure/explorer-backend",
}
```

Comment (tdroxler) [reply docusaurus.config.js:216]: ooooooooh ok, my bad

Comment (tdroxler) [reply docusaurus.config.js:216]: @nop33 fix pushed

**Comment by tdroxler** [comment]
@polarker @nop33 anything else to fix?

File: docs/infrastructure/explorer-backend.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+sidebar_position: 15
+title: Explorer Backend
+sidebar_label: Explorer backend
+---
+
+Alephium's explorer backend is an indexer that provides a RESTful API to query the Alephium blockchain.
+
+It serves https://explorer.alephium.org/ as well as our wallets.
+
+Find more information in the [Alephium Explorer Backend repository](https://github.com/alephium/explorer-backend/)
```
Comment (nop33) [inline docs/infrastructure/explorer-backend.md]: Missing dot at the end `.`

File: docs/infrastructure/explorer-backend.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+   postgres=# CREATE DATABASE explorer;
+   ```
+
+## Start your explorer-backend
+
+```shell
+java -jar explorer-backend-x.x.x.jar
+```
+
+Your explorer-backend will start to sync with the full node. It might take long the first time or start from a snapshot.
```
Comment (nop33) [inline docs/infrastructure/explorer-backend.md]: I would rephrase this to

```suggestion
Your explorer-backend will start to sync with the full node. When syncing for the first time the process might take long. To speed it up you can [start from a snapshot](#start-from-a-snapshot).
```

File: docs/infrastructure/explorer-backend.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+
+## Configuration
+
+The default configuration is available in [application.conf](https://github.com/alephium/explorer-backend/blob/master/app/src/main/resources/application.conf).
+
+Everything can be override with two ways:
```
Comment (nop33) [inline docs/infrastructure/explorer-backend.md]:
```suggestion
Everything can be overridden in two ways:
```

File: docs/infrastructure/explorer-backend.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+### `user.conf` file
+
+You can change the config with the `~/.alephium-explorer-backend/user.conf` file. e.g:
```
Comment (nop33) [inline docs/infrastructure/explorer-backend.md]:
```suggestion
You can change the config in the `~/.alephium-explorer-backend/user.conf` file. e.g:
```

File: docs/infrastructure/explorer-backend.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+- Java (11 or 17 is recommended)
+- [PostgreSQL](https://www.postgresql.org)
+- A running [full node](full-node/getting-started.md)
+
+## Download Application File
+
+Download file `explorer-backend-x.x.x.jar` from [Github release](https://github.com/alephium/explorer-backend/releases/latest).
+
+## Create the database:
```
Comment (nop33) [inline docs/infrastructure/explorer-backend.md]: nit: Remove `:` or add them in every header

File: docs/infrastructure/explorer-backend.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+
+```conf
+alephium {
+  explorer {
+      port = 9191 //Change default 9090 port
+  }
+}
+```
+
+### Environment variables:
```
Comment (nop33) [inline docs/infrastructure/explorer-backend.md]: nit: Remove `:` or add them in every header

File: docs/infrastructure/explorer-backend.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+```
+
+## Start from a snapshot
+
+Alephium [archives repository](https://archives.alephium.org) also contain the snapshots for explorer backend database.
+The snapshot can be loaded in the postgresql database of the explorer backend at the first run, using the command below.
+
+* Make sure to use the network you want to load the snapshot for, and the correct database name and user.
+* The database must be created before running the command and must be empty.
```
Comment (nop33) [inline docs/infrastructure/explorer-backend.md:88]:
```suggestion
:::warning

* Make sure to use the network you want to load the snapshot for, and the correct database name and user.
* The database must be created before running the command and must be empty.

:::
```

This will render a nice box:

image

**Review by nop33 — COMMENTED** [review]
👏

**Comment by tdroxler** [comment]
can I merge this?

**Comment by polarker** [comment]
Sorry, I missed this. Let's merge it!
